public/coffee/ide/editor/directives/aceEditor/spell-check/SpellCheckManager.coffee

```diff
@@ -246,3 +250,28 @@ define [
 				error: (xhr, status, error) ->
 					callback error
 			return $.ajax options
+
+		blacklistedCommandRegex: ///
+			\\                    # initial backslash
+			(label                # any of these commands
+			|[a-z]{0,2}ref
+			|usepackage
+			|begin
+			|end
+			|[a-z]{0,2}cite
```

Comment:
There are `\...cite` commands with more than two prefix letters, e.g. `\parencite`. I'd imagine similarly for `\ref`. Is the concern about performance/scanning far ahead? If so, just something big like 16?